contributing/community/speaker-mentoring.rst

```diff
@@ -0,0 +1,43 @@
+Speaker mentoring
+=================
+
+The Symfony community benefits greatly from as many people as possible
+sharing their knowledge and experience with others. Every different
+point of view adds to our collective understanding of how to best use
+and evolve the code, design patterns and architecture provided within
+the Symfony community. As such we specifically welcome not only sharing
```

Comment:
> ~~As such~~ Because of this, we specifically want to hear from both long-time contributors *and* new users, who often come across entirely different challenges and have a totally fresh look and perspective.